### ISSUE TITLE
Problems using lib in simulator

### ISSUE THREAD

**Issue by rafagan** [ibireme/YYImage#162]
The library is presenting the following issue when build for iOS simulator:

In <...>/Pods/YYImage/Vendor/WebP.framework/WebP(libwebpencode_la-config.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64

Any help?

**Comment by GiuseppePiscopo** [comment]
Hi there, we're having the same issue, with manual installation as we manage dependencies through Carthage:

> [omitted]/Vendor/WebP.framework/WebP(libwebpencode_la-config.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

This apparently only happens when building in Release configuration for a simulator destination. It does not happen in the remaining three cases (Debug for simulator and bot Release/Debug for device).

Did you have any luck in solving the issue?  
Thanks

**Comment by rafagan** [comment]
No, I don't. For now I'm using only real devices. I think this library wasn't configured for x86_64 architectures

**Comment by GiuseppePiscopo** [comment]
Thanks for reply. In the meantime we found the issue. There are a couple of workarounds (not definitive solutions, I mean). You can find more details in a (quite dense) Stack Overflow page: https://stackoverflow.com/questions/63607158/xcode-12-building-for-ios-simulator-but-linking-in-an-object-file-built-for-io/

I'd suggest to especially read the current second answer for an understanding of the issue: https://stackoverflow.com/questions/63607158/xcode-12-building-for-ios-simulator-but-linking-in-an-object-file-built-for-io/64139830#64139830

The two workarounds are the one related to `ONLY_ACTIVE_ARCH` and the one related to `EXCLUDED_ARCHS`.  
HTH

**Comment by GiuseppePiscopo** [comment]
@rafagan you might want to consider closing the issue, if that's the case. Regards
